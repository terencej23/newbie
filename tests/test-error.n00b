& = "this will give an error"
