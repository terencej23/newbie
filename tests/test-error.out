Uncaught exception: Exceptions.SyntaxError("unrecognized char '&'")
Fatal error: exception Exceptions.SyntaxError("unrecognized char '&'")